ui/src/components/Bridge.js

```diff
@@ -37,7 +37,8 @@ export class Bridge extends React.Component {
 
   handleInputChange = name => event => {
     this.setState({
-      [name]: event.target.value
+      [name]: event.target.value,
+      "formattedBalance": event.target.value
```

Comment:
please remove comma separator as well @bajaj6390ankit since it cannot trigger the 'Transfer'